packages/api/src/platforms/vtex/resolvers/query.ts

```diff
@@ -172,4 +172,16 @@ export const Query = {
       }
     )
   },
+  topSearches: async (_: unknown, __: unknown, ctx: Context) => {
+    const {
+      clients: { search },
+    } = ctx
+
+    const { searches } = await search.topSearches()
+
+    return searches.map((searchTermInfo) => ({
```

Comment:
This result is ordered? wdyt about adding a way to order this result?

Reply:
I have to confirm this, but I'm pretty sure the API returns the results ordered from most to least searched.

Reply:
About adding a way to control the order... we could add it, but first let me check if the API supports it

Reply:
Just checked with the IS team, this API doesn't support any configuration for now, so I guess it's fine for now to not add any arguments to this query.